src/components/Layout.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import Navbar from './Navbar';
+
+const Layout = () => (
+  // <div className="layout">
```

Comment:
- [ ] Kindly avoid leaving commented pieces of code in your codebase. It makes your codebase unclean and is not a good practice :+1: